modules/authentication/proc-enabling-authentication-with-github.adoc

```diff
@@ -93,13 +93,52 @@ signInPage: github # <3>
 <2> Apply the GitHub credentials configured in your {product-short} secrets.
 <3> To enable the GitHub provider as your {product-short} sign-in provider.
 
-.. Optional: Consider adding the following optional fields:
+`environment: production`::
+Mark the environment as `production` to hide the Guest login in the {product-short} home page.
+
+`clientId`, `clientSecret`, `host`, `appId`, `webhookUrl`, `webhookSecret`, `privateKey`::
+Use the {product-short} application information that you have created in GitHub and configured in OpenShift as secrets.
+
+`sigInPage: github`::
+To enable the GitHub provider as default sign-in provider.
+
+Optional: Consider adding the following optional fields:
+
+`dangerouslyAllowSignInWithoutUserInCatalog: true`::
+To enable authentication without requiring to provision users in the {product-short} software catalog.
++
+WARNING: Use `dangerouslyAllowSignInWithoutUserInCatalog` to explore {product-short} features, but do not use it in production.
++
+.`{my-app-config-file}` fragment with optional field to allow authenticating users absent from the software catalog
+[source,yaml]
+----
+auth:
+  environment: production
+  providers:
+    github:
+      production:
+        clientId: ${AUTH_GITHUB_CLIENT_ID}
+        clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
+integrations:
+  github:
+    - host: ${GITHUB_HOST_DOMAIN}
+      apps:
+        - appId: ${AUTH_GITHUB_APP_ID}
+          clientId: ${AUTH_GITHUB_CLIENT_ID}
+          clientSecret: ${GITHUB_CLIENT_SECRET}
+          webhookUrl: ${GITHUB_WEBHOOK_URL}
+          webhookSecret: ${GITHUB_WEBHOOK_SECRET}
+          privateKey: |
+            ${GITHUB_PRIVATE_KEY_FILE}
+signInPage: github
+dangerouslyAllowSignInWithoutUserInCatalog: true
+----
```

Comment:
This is unfortunately reverting changes made by https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/982/files

Reply:
```suggestion
```